.github/workflows/tests.yml

```
@@ -43,7 +45,25 @@ jobs:
         done
 
     - name: Check for CI-syntax-only label
-      id: check-labels
+      id: skip-arm-label
+      uses: actions/github-script@v3
+      if: github.event_name == 'pull_request'
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        script: |
+          const { data: { labels: labels } } = await github.pulls.get({
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            pull_number: context.issue.number
+          })
+          if (labels.map(label => label.name).includes('CI-skip-arm')) {
+            console.log('CI-skip-arm label found.')
+            return true
+          }
+          console.log('No CI-skip-arm label found.')
+
+    - name: Check for CI-skip-arm label
+      id: skip
```

Maybe:

```suggestion
      id: skip-tests-label
```

(Needs to match the entry in `outputs`)